### PR TITLE
Use wl protocol for keeping it centered

### PR DESCRIPTION
**Pull request by leolost2605**
Goes with elementary/gala#1942

### CODE REVIEW

File: src/PolkitDialog.vala [hunk 1]
```diff
@@ -123,6 +125,18 @@ public class Ag.PolkitDialog : Granite.MessageDialog {
...
+
+        var gesture_click = new Gtk.GestureClick () {
+            propagation_phase = CAPTURE
+        };
+        child.add_controller (gesture_click);
+        gesture_click.pressed.connect (() => {
+            if (extended_behavior != null) {
+                extended_behavior.focus ();
+            }
+        });
```
Comment (danirabbit) [inline src/PolkitDialog.vala:137]: Why is this on click?

**Review by danirabbit — COMMENTED** [review]
We need to https://github.com/elementary/gala/pull/1983 for this to work right?

Also one question

**Comment by tintou** [comment]
Note that this is still not a "widget" :wink:

**Review by danirabbit — APPROVED** [review]
LFG :rocket:

**Pull request merged**